api/proto/config/devmodel.proto

```diff
@@ -131,6 +142,8 @@ message PhysicalIO {
       // "ioports": the address is a string such as "2f8-2ff"
       map <string, string> phyaddrs = 3;
 
+      // logicallabel -- Should this move to System Adapters? This can keep
```

Comment:
I think there are multiple levels of logical labels in the fullness of time.
The PhyIO has a physical label which is something printed on the unit.
It is probably convenient to be able to specify a logical label (such as "eth0") for it.
But then one might want a L2 model which specifies to take eth0+eth3 and create a LAG/bond, assigning the logical label bond0 to that construct.
Then a second L2 model which specifies that .1Q tag 10 on "bond0" show be created and have a logical label "shopfloor-east". 

The system adapter will merely refer to the logical level in the L2 model.
Hence I don't think this needs to be moved.